.github/workflows/ci.yml

```
@@ -31,6 +31,9 @@ jobs:
 
             - name: Prepare block-meta files
```

```suggestion
            - name: Copy schema files
```

This jobs copies the GraphQL `schema.gql` file as well.